libs/framework/src/framework.c

```diff
@@ -1697,7 +1761,7 @@ celix_status_t framework_markBundleResolved(framework_pt framework, module_pt mo
             }
 
             status = CELIX_DO_IF(status, framework_setBundleStateAndNotify(framework, bundle, OSGI_FRAMEWORK_BUNDLE_RESOLVED));
-            status = CELIX_DO_IF(status, fw_fireBundleEvent(framework, OSGI_FRAMEWORK_BUNDLE_EVENT_RESOLVED, bundle));
+            status = CELIX_DO_IF(status, fw_fireBundleEvent(framework, OSGI_FRAMEWORK_BUNDLE_EVENT_RESOLVED, entry));
         }
 
         if (status != CELIX_SUCCESS) {
```

Comment:
Can you take another look at the FIXME on line 1787?

Reply:
I will deprecate the function